forch/authenticator.py

```diff
@@ -133,6 +134,8 @@ def parse_args(raw_args):
                         help='RADIUS server ip')
     parser.add_argument('-p', '--server-port', type=int, default=1812,
                         help='Server port that freeradius is listening on')
+    parser.add_argument('-l', '--listen-port', type=int, default=0,
```

Comment:
How is this different than the previous line?  Server ports are typically listen ports?  Is the term "freeradius" in the previous one wrong?  Shouldn't that be "Remote radius server port"?

Reply:
Freeradius is the process that radius servers run. Should mean the same as remote RADIUS server but I can change it.